config/dev.exs

```diff
@@ -51,8 +51,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :jeopardy, Jeopardy.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "Pa$$word",
+  username: "tasktracer",
+  password: "aBc123$",
```

Comment:
I think we should have the dev database username be 'jeopardy' or something (I think we've all been changing it to different things so we should pick something and keep it consistent, so everyone can set up their postgres stuff locally once and not have to change it)